utils/config.go

```diff
@@ -13,9 +13,9 @@ import (
 	"time"
 )
 
-var LibexecDir = "/usr/local/libexec"
-var EtcDir = "/usr/local/etc"
-var DataDir = "/usr/local/share"
+var LibexecDir = "."
```

Comment:
What's the reason to change the directory names back to "."?

Reply:
okay i saw your commit "Set DataDir, EtcDir, etc. to "." by default so that GSKY can be run"